backend/app/notify/email.go

```diff
@@ -241,14 +209,19 @@ func (e *Email) buildMessageFromRequest(req Request) (string, error) {
 		return "", errors.Wrapf(err, "error creating token for unsubscribe link")
 	}
 	msg := bytes.Buffer{}
-	err = e.msgTmpl.Execute(&msg, msgTmplData{
-		CommentUser:     req.Comment.User.Name,
-		ParentUser:      req.parent.User.Name,
-		CommentText:     req.Comment.Text,
-		CommentLink:     req.Comment.Locator.URL + uiNav + req.Comment.ID,
-		PostTitle:       req.Comment.PostTitle,
-		Email:           req.Email,
-		UnsubscribeLink: unsubscribeLink,
+	err = e.replyTmpl.Execute(&msg, replyTmplData{
+		UserName:          req.Comment.User.Name,
+		UserPicture:       req.Comment.User.Picture,
+		CommentText:       req.Comment.Text,
+		CommentLink:       req.Comment.Locator.URL + uiNav + req.Comment.ID,
```

Comment:
I would suggest to move `req.Comment.Locator.URL + uiNav` to variable with name something like `commentUrlPrefix`